### PR TITLE
adding ECDSA AKD + presignatures post :)

### PR DESCRIPTION
**Pull request by rbkhmrcr**

### CODE REVIEW

File: blogo-input/posts/ecdsa-akd.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
+## rwc2022 : Threshold ECDSA with additive key derivation and presignatures : an attack, and a solution
+
```
Comment (arnaucube) [inline blogo-input/posts/ecdsa-akd.md:2]: Here there could be added the date & name:
```suggestion
*2022-05-11 by [Name](https://link-to-web/git/twitter/blog/etc)*
```

**Review by arnaucube — COMMENTED** [review]
Nice!! :cat: 
One small comment, there are lot of math expressions like \`K = kG\` in the post, if you want you can use $K = kG$ to use latex features of the blog.

**Comment by rbkhmrcr** [comment]
I thought about latex vs code boxes but I like the code boxes because they break the flow of the text less! or did you find it hard to follow what's in the code boxes?

**Review by arnaucube — APPROVED** [review]
